kinto/core/resource/schema.py

```diff
@@ -150,3 +151,71 @@ class BookmarkSchema(ResourceSchema):
 
     def preparer(self, appstruct):
         return strip_whitespace(appstruct)
+
+
+class CSVQuerystring(colander.SchemaNode):
+    """String field representing a list of attributes."""
+
+    schema_type = colander.String
+    missing = colander.drop
+    error_message = "The value should be a list of comma separated field names"
+
+    def deserialize(self, cstruct=colander.null):
+        params = super(CSVQuerystring, self).deserialize(cstruct)
+        if params is colander.drop:
+            return params
+        else:
+            return params.split(',')
```

Comment:
I think there is the notion of `preparer` in Colander that could split, and then the notion of `Sequence`. In a second iteration, we can try to leverage that if possible